ignite/services/chain/build.go

```diff
@@ -204,7 +205,7 @@ func (c *Chain) preBuild(ctx context.Context, cacheStorage cache.Storage) (build
 		gocmd.FlagLdflags, gocmd.Ldflags(ldFlags...),
 	}
 
-	c.ev.Send("📦 Installing dependencies...")
+	c.ev.Send("Installing dependencies...", events.ProgressUpdate())
```

Comment:
Why are we removing the emojis?

Reply:
I left a [comment](https://github.com/ignite/cli/pull/2957#pullrequestreview-1148869155) in another issue which explains the reason behind removing the icon when using the spinner status text.

In a nut-shell, using the event bus with the `ProgressStart` or `ProgressUpdate` option will show/update a blue spinning icon and a text, where both are removed once the session ends or a `ProgressFinish` event is received. Using this functionality with a prefix icon would result in "SPINER ICON TEXT" which, arguably, doesn't "look good".

I can restore them in any case :).

Is good to mention that it would be good to define a "standard" for the CLI output or temporary status texts when using the event bus so we have a consistent user experience. I created another [issue](https://github.com/ignite/cli/issues/2967) that is related to the introduction of the event bus and is related to this.